wrappers/javascript/indy-vdr-nodejs/README.md

```diff
@@ -51,3 +51,13 @@ yarn test:local-build
 ```
 
 > **Note**: If you want to use this library in a cross-platform environment you need to import methods from the `@hyperledger/indy-vdr-shared` package instead. This is a platform independent package that allows to register the native bindings. The `@hyperledger/indy-vdr-nodejs` package uses this package under the hood. See the [Indy VDR Shared README](https://github.com/hyperledger/indy-vdr/tree/main/wrappers/javascript/indy-vdr-shared/README.md) for documentation on how to use this package.
+
+## Version Compatibility
+
+The JavaScript wrapper is versioned independently from the native bindings. The following table shows the compatibility between the different versions:
+
+| Indy VDR      | JavaScript Wrapper |
+| ------------- | ------------------ |
+| v0.4.0-dev.16 | v0.1.0             |
+| v0.4.1        | v0.2.0             |
+| v0.4.2        | v0.2.3             |
```

Comment:
I think it will be clearer if we explicitly mention that v0.4.1 corresponds to 0.2.0, 0.2.1 and 0.2.2, or just 0.2.x and call this one (the one under development) as 0.3.0.

Actually in any case I'd prefer to move on to 0.3.0